tests/continuous/test_nse.py

```diff
@@ -0,0 +1,71 @@
+import numpy as np
+import pandas as pd
+import pytest
+import xarray as xr
+
+import scores
+from scores.continuous.standard_impl import nse
+
+
+def test_xarray_dataarray():
+    fcst_xr = xr.DataArray([3, 4, 5, 6, 7])
+    obs_xr = xr.DataArray([2, 3, 4, 5, 6])
+    assert nse(fcst_xr, obs_xr) == 0.5
+
+
+def test_nse_with_datasets():
+    # Create dataset
+    time = pd.date_range("2024-01-01", periods=5, freq="D")
+    stations = ["Station1", "Station2", "Station3"]
+    fcst_data = np.array([[3, 4, 5, 6, 7], [3, 4, 5, 6, 7], [3, 4, 5, 6, 7]])
+    obs_data = np.array([[2, 3, 4, 5, 6], [2, 3, 4, 5, 6], [2, 3, 4, 5, 6]])
+    fcst_xr = xr.DataArray(fcst_data, coords={"time": time, "station": stations}, dims=["station", "time"])
+    obs_xr = xr.DataArray(obs_data, coords={"time": time, "station": stations}, dims=["station", "time"])
+
+    fcst_ds = xr.Dataset({"forecast": fcst_xr})
+    obs_ds = xr.Dataset({"observed": obs_xr})
```

Comment:
Can you please add tests for `xr.Dataset`s with multiple data vars (e.g., river height, and flow rate)